core/XH.js

```diff
@@ -305,13 +305,16 @@ class XHClass {
      * @param {string} config.message - message text to be displayed.
      * @param {string} [config.title] - title of message box.
      * @param {Element} [config.icon] - icon to be displayed.
+     * @param {MessageInput|boolean} [config.input] - config for input to be displayed, or true to use default TextInput.
```

Comment:
wouldn't it be more straightforward to have .message() just require a MessageInput here?   Leave the defaulting to .prompt.  Message() is the full, explicit form